### ISSUE TITLE
Apps can't be managed when circles app is active: Safari

### ISSUE THREAD

**Issue by skerbis** [nextcloud/circles#1528]
When I click on an App to manage it in Safari while circles is activated, 
it is not possible to reach the sidebar for it. 

Here I cliced on Calendar. But it appears Two-Factor TOTP-Provider in the sidebar. 
Allways it also happens when I click on a different App.

<img width="1326" alt="Bildschirmfoto 2024-03-14 um 10 30 08" src="https://github.com/nextcloud/circles/assets/791247/3b5b8d21-6e9b-4476-b3ad-0902b1ad9dfa">

**Comment by joshtrichards** [comment]
Are you still experiencing this? I believe this was a Server matter a few maintenance releases back, not the Circles app.

**Comment by skerbis** [comment]
yes. But it seems not to be cirlces' fault. It seems to be an issue with collectives app. When I disable it, everything is fine.
